### PR TITLE
possibility to add window target in client

### PR DESCRIPTION
**Pull request by Bettelstab**

### CODE REVIEW

File: client/KeyguardClient.js [hunk 1]
```diff
@@ -129,9 +136,11 @@ class KeyguardClient {
      * @param {any[]} [args]
      */
     async _startPopup(requestName, args) {
+        const requestUrl = `${this._keyguardSrc}/request/${requestName}/`;
+
         const $popup = window.open(
-            `${this._keyguardSrc}request/${requestName}/`,
-            'Nimiq Keyguard Popup',
+            requestUrl,
+            this._targetName,
```
Comment (danimoh) [inline client/KeyguardClient.js]: Mdn says `The name should not contain whitespace.` (https://developer.mozilla.org/en-US/docs/Web/API/Window/open)

Comment (svub) [reply client/KeyguardClient.js]: yep, I remember it breaks in some browsers if you do

Comment (svub) [reply client/KeyguardClient.js]: the `name` is the identifier for this "frame" - the window title will be set by the HTML page loaded inside.

File: src/common.js [hunk 2]
```diff
@@ -37,6 +37,4 @@ async function runKeyguard(RequestApiClass, options) { // eslint-disable-line no
```
Comment (svub) [inline src/common.js]: Hasn't `I18n` been moved to PopupAPI?

Comment (sisou) [reply src/common.js]: That PR is still pending (#48).

**Pull request merged**